tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -847,6 +847,108 @@ Valid types are:
     You can turn off this check, or make it only a warning, by changing it in
     the project settings. See :ref:`doc_gdscript_warning_system` for details.
 
+Static variables
+^^^^^^^^^^^^^^^^
+
+A class member variable can be declared static::
+
+    static var a
+
+The ``@export`` and ``@onready`` annotations cannot be applied to a static variable.
```

Comment:
Does it make sense to move these two lines after the static variable explanation (currently line number 866)? I believe that is better flow of information. Also we can make it as warning or tip?